packages/server/src/queue/queue.module.ts

```diff
@@ -13,8 +15,9 @@ import { QueueSubscriber } from './queue.subscriber';
     QueueService,
     QueueSSEService,
     QueueSubscriber,
+    AlertsService,
   ],
   exports: [QueueCleanService, QueueSSEService],
-  imports: [SSEModule],
+  imports: [SSEModule, AlertsModule],
```

Comment:
im boomer, `AlertsModule` needs to be added/kept here right? for dependencies and stuff

Reply:
oh i see the problem

Reply:
I think so? I forget bc it's been a while since I've used worked with these

Reply:
yea i think we do in order to inject the `AlertsService` I added